src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public interface Drawable {
+    String drawFigure();
+
+    double getArea();
+
+    String getUniqueProperties();
```

Comment:
Is it correct to add these methods to this interface? They relate to drawing. I recommend creating another interface for calculating the area. And you don`t need method getUniqueProperties()